soc/ti/k3/soc.yml

```diff
@@ -3,6 +3,9 @@ family:
   series:
   - name: am6x
     socs:
+    - name: am6232
```

Comment:
If they must all be enumerated, am6254 should be one of the options. I wonder if there is a way to simply leave it off of M4-based entries, but certainly it is needed for A53-based entries to know the number of cores.